### ISSUE TITLE
Allow hiding OrderBy options

### ISSUE THREAD

**Issue by hapoza** [vtex-apps/store-discussion#101]
**Is your feature request related to a problem? Please describe.**
We are working with https://biggy.io/ on creating their search app which only replaces the search page context, and accepts a search-result block which then uses the OrderBy component that has hardcoded options. Their system does not support `OrderByName` options so we need to disable them. 

**Describe the solution you'd like**
A `hiddenFacets.orderBy` prop which allows disabling some of those options. Similar to how it's already possible to hide specificationFilters 

**Describe alternatives you've considered**
None

**Additional context**
https://github.com/vtex-apps/search-result/blob/master/react/OrderBy.js#L9
https://github.com/vtex-apps/search-result#hiddenfacets

**Comment by klzns** [comment]
Can it be achieved using CSS?

**Comment by hapoza** [comment]
There is no class that differentiates one item from another:
`<button class="vtex-search-result-3-x-orderByOptionItem hover-bg-muted-5 bg-base  c-on-base f5 ml-auto db no-underline pointer tl bn pv4 ph5 w-100 right-0-ns">De A a Z</button>`

We could do it using `nth-child` selectors, but that is a fragile selector and prone to breaking on the future when vtex.search-result updates.

**Comment by denisebiscaro** [comment]
That's a good first issue, basically add the a class appended with the value here:
https://github.com/vtex-apps/search-result/blob/master/react/components/SelectionListOrderBy.js#L47-L54

**Comment by hapoza** [comment]
@denisebiscaro What you showed doesn't differentiate each filter from it's siblings. I did it with a hacky workaround using CSS, something like what was done on #5 would help.
And I noticed another problem, those filters that I've hidden only with CSS show up as options on Site Editor, so it would be possible for someone managing the store content to choose those options as the default sorting method and notice that they don't work.

I'll leave it up to you to close the issue or not, I was able to do what we needed with the CSS workaround, but maybe something like this will pop up again in the future when integrating other search recommendation systems like Biggy.
